apps/dashboard/src/contract-ui/tabs/claim-conditions/components/claim-conditions-form/phase.tsx

```diff
@@ -157,11 +156,6 @@ export const ClaimConditionsPhase: React.FC<ClaimConditionsPhaseProps> = ({
             claimConditionType === "creator" ? null : (
               <CustomFormGroup>
                 <MaxClaimablePerWalletInput />
```

Comment:
what does this change do visually? we used to still render the empty `Box` here so is the input that's left now too big?

Reply:
ty ser